Leetcode/EigenDemo/eigenPractice.cpp

```diff
@@ -0,0 +1,103 @@
+//
+//  eigenPractice.cpp
+//  EigenDemo
+//
+//  Created by dsh on 2021/7/18.
+//
+#include "eigenPractice.hpp"
+
+void eigenPractice::eigenBasic() {
+    MatrixXd m(2,2);
+    m(0,0) = 3;
+    m(1,0) = 2.5;
+    m(0,1) = -1;
+    m(1,1) = m(1,0) + m(0,1);
+    std::cout << m << std::endl;
+
+    Matrix<float, 2, 3> matrix_23;
+    Vector3d v_3d;
+    Matrix<float, 3, 1> vd_3d;
+    Matrix3d matrix_33 = Matrix3d::Zero();
+    Matrix<double, Dynamic, Dynamic> matrix_dynamic;
+    MatrixXd matrix_x;
+
+    matrix_23 << 1, 2, 3, 4, 5, 6;
+    std::cout << "matrix 2x3 : \n" << matrix_23 << std::endl;
+    std::cout << "print matrix 2x3: " << std::endl;
+    for (int i = 0; i < 2; i++) {
+        for (int j = 0; j < 3; j++) std::cout << matrix_23(i, j) << "\t";
```

Comment:
```suggestion
        for (int j = 0; j < 3; j++)
            std::cout << matrix_23(i, j) << "\t";
```

https://google.github.io/styleguide/cppguide.html#Conditionals
https://google.github.io/styleguide/cppguide.html#Loops_and_Switch_Statements

不太建议单行写完。